backend/src/model/entity/plants_impl.rs

```diff
@@ -36,6 +33,10 @@ impl Plants {
         page_parameters: PageParameters,
         conn: &mut AsyncPgConnection,
     ) -> QueryResult<Page<PlantsSummaryDto>> {
+        let set_similarity_threshold = diesel::sql_query("SET pg_trgm.similarity_threshold=0.1");
```

Comment:
We might want to set this as an environment variable in the backend.
According to pg_trgm's documentation this can only be set for a specific query or for an entire session, but not for the entire database.

Reply:
No, it is fine to hard-code this.

Reply:
Let us try to have this in migrations. But it is also fine like it is (the same connection is used, so it should be super-fast).